.buildkite/pipeline.yml

```diff
@@ -31,13 +31,9 @@ steps:
   #################
   - label: "🧹 Lint"
     key: "lint"
-    command: .buildkite/lint.sh
-    plugins:
-      - *bash_cache
-      - docker#v3.8.0:
-          image: "public.ecr.aws/automattic/multilint:latest"
-    agents:
-      queue: "default"
+    command: "lint_pod"
+    env: *common_env
+    plugins: *common_plugins
```

Comment:
I moved this back to the macOS hosts because docker images and library plugins don't play nicely together – I have a workaround, but it'd require a new docker image, so this was a faster way to prove that this works (also it might've been an over-optimization...)